### ISSUE TITLE
Error when i tried build using Cargo

### ISSUE THREAD

**Issue by AndresD172** [Toqozz/wired-notify#51]
Today I tried to compile Wired Notify following the instructions in the readme file, but when I ran `cargo build --release` Cargo gave me the following error: 
![2022-02-18-210724_1226x169_scrot](https://user-images.githubusercontent.com/57467120/154783784-27d0d8e9-88b8-4d02-9d10-c4215f2095da.png)
 and I cannot continue the build, can someone help me with my problem?

Thx <3!

**Comment by AndresD172** [comment]
Ah, I forgot to indicate the distro I'm using! It's Fedora 35, and in theory I have x11 installed, I even use several x11 tools in my own scripts.

**Comment by Toqozz** [comment]
You may need the developer library -- `libxss-dev`.  Not sure what the exact name for the package is on Fedora, but try it.

**Comment by AndresD172** [comment]
Hi @Toqozz! After a week of testing and a kernel corruption (no kidding, I corrupted the kernel trying to install pango) I've finally been able to make the wired-notify build. To do this I had to install a set of Rust integrations, using the command `sudo dnf install rust-x11+xss-devel.noarch rust-glib+v2_68-devel.noarch dbus-devel pkgconf-pkg-config rust-pango+default-devel.noarch rust-cairo-rs+default-devel.noarch`

I also intend to create some kind of installer with Bash for RHEL-based distributions, since I guess that similar problems with compilation also occur in other distributions. I really don't know if I'm going to make it happen though. 

So if you want me to keep you up to date with the installer contact me on Discord Andres Chacon#9220!

Greetings! <3
